Description: Upstream changes introduced in version 2.2+git20110628-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 lcms2 (2.2+git20110628-2) unstable; urgency=low
 .
   * debian/control:
     - Updated libjpeg dependency for the transition.
     - Updated package descriptions for better search results.
 .
 The person named in the Author field signed this changelog entry.
Author: Oleksandr Moskalenko <malex@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- lcms2-2.2+git20110628.orig/Makefile
+++ lcms2-2.2+git20110628/Makefile
@@ -128,13 +128,13 @@ DIST_ARCHIVES = $(distdir).tar.gz $(dist
 GZIP_ENV = --best
 distuninstallcheck_listfiles = find . -type f -print
 distcleancheck_listfiles = find . -type f -print
-ACLOCAL = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -166,7 +166,7 @@ INSTALL_STRIP_PROGRAM = $(install_sh) -c
 JPEGICC_DEPLIBS = -ljpeg -lm
 LCMS_LIB_DEPLIBS = -lm
 LD = /usr/bin/ld -m elf_x86_64
-LDFLAGS = 
+LDFLAGS = -Wl,-Bsymbolic-functions
 LIBOBJS = 
 LIBRARY_AGE = 0
 LIBRARY_CURRENT = 2
@@ -182,7 +182,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -206,10 +206,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
-abs_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
-abs_top_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
-abs_top_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
+abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -240,7 +240,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
--- lcms2-2.2+git20110628.orig/utils/linkicc/Makefile
+++ lcms2-2.2+git20110628/utils/linkicc/Makefile
@@ -76,13 +76,13 @@ DIST_SOURCES = $(linkicc_SOURCES)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -114,7 +114,7 @@ INSTALL_STRIP_PROGRAM = $(install_sh) -c
 JPEGICC_DEPLIBS = -ljpeg -lm
 LCMS_LIB_DEPLIBS = -lm
 LD = /usr/bin/ld -m elf_x86_64
-LDFLAGS = 
+LDFLAGS = -Wl,-Bsymbolic-functions
 LIBOBJS = 
 LIBRARY_AGE = 0
 LIBRARY_CURRENT = 2
@@ -130,7 +130,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -154,10 +154,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/utils/linkicc
-abs_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/utils/linkicc
-abs_top_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
-abs_top_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
+abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/linkicc
+abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/linkicc
+abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -188,7 +188,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
@@ -215,7 +215,7 @@ AM_CPPFLAGS = -I$(top_builddir)/include
               -I$(top_srcdir)/utils/common -I$(top_builddir)/utils/common
 
 linkicc_LDADD = $(top_builddir)/src/liblcms2.la 
-linkicc_LDFLAGS = 
+linkicc_LDFLAGS = -Wl,-Bsymbolic-functions
 linkicc_SOURCES = linkicc.c ../common/xgetopt.c ../common/vprf.c ../common/utils.h
 linkicc_MANS = linkicc.1
 EXTRA_DIST = $(man_MANS)
--- lcms2-2.2+git20110628.orig/utils/tificc/Makefile
+++ lcms2-2.2+git20110628/utils/tificc/Makefile
@@ -99,13 +99,13 @@ MANS = $(man_MANS)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -137,7 +137,7 @@ INSTALL_STRIP_PROGRAM = $(install_sh) -c
 JPEGICC_DEPLIBS = -ljpeg -lm
 LCMS_LIB_DEPLIBS = -lm
 LD = /usr/bin/ld -m elf_x86_64
-LDFLAGS = 
+LDFLAGS = -Wl,-Bsymbolic-functions
 LIBOBJS = 
 LIBRARY_AGE = 0
 LIBRARY_CURRENT = 2
@@ -153,7 +153,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -177,10 +177,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/utils/tificc
-abs_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/utils/tificc
-abs_top_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
-abs_top_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
+abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/tificc
+abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/tificc
+abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -211,7 +211,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
@@ -238,7 +238,7 @@ AM_CPPFLAGS = -I$(top_builddir)/include
               -I$(top_srcdir)/utils/common -I$(top_builddir)/utils/common
 
 tificc_LDADD = $(top_builddir)/src/liblcms2.la -ltiff -ljpeg -lz -lm
-tificc_LDFLAGS = 
+tificc_LDFLAGS = -Wl,-Bsymbolic-functions
 tificc_SOURCES = tificc.c ../common/xgetopt.c ../common/vprf.c ../common/utils.h
 man_MANS = tificc.1
 EXTRA_DIST = $(man_MANS)
--- lcms2-2.2+git20110628.orig/utils/transicc/Makefile
+++ lcms2-2.2+git20110628/utils/transicc/Makefile
@@ -76,13 +76,13 @@ DIST_SOURCES = $(transicc_SOURCES)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -114,7 +114,7 @@ INSTALL_STRIP_PROGRAM = $(install_sh) -c
 JPEGICC_DEPLIBS = -ljpeg -lm
 LCMS_LIB_DEPLIBS = -lm
 LD = /usr/bin/ld -m elf_x86_64
-LDFLAGS = 
+LDFLAGS = -Wl,-Bsymbolic-functions
 LIBOBJS = 
 LIBRARY_AGE = 0
 LIBRARY_CURRENT = 2
@@ -130,7 +130,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -154,10 +154,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/utils/transicc
-abs_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/utils/transicc
-abs_top_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
-abs_top_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
+abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/transicc
+abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/transicc
+abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -188,7 +188,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
@@ -215,7 +215,7 @@ AM_CPPFLAGS = -I$(top_builddir)/include
                -I$(top_srcdir)/utils/common -I$(top_builddir)/utils/common
 
 transicc_LDADD = $(top_builddir)/src/liblcms2.la 
-transicc_LDFLAGS = 
+transicc_LDFLAGS = -Wl,-Bsymbolic-functions
 transicc_SOURCES = transicc.c ../common/xgetopt.c ../common/vprf.c  ../common/utils.h
 transicc_MANS = transicc.1
 EXTRA_DIST = $(man_MANS)
--- lcms2-2.2+git20110628.orig/utils/jpgicc/Makefile
+++ lcms2-2.2+git20110628/utils/jpgicc/Makefile
@@ -100,13 +100,13 @@ MANS = $(man_MANS)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -138,7 +138,7 @@ INSTALL_STRIP_PROGRAM = $(install_sh) -c
 JPEGICC_DEPLIBS = -ljpeg -lm
 LCMS_LIB_DEPLIBS = -lm
 LD = /usr/bin/ld -m elf_x86_64
-LDFLAGS = 
+LDFLAGS = -Wl,-Bsymbolic-functions
 LIBOBJS = 
 LIBRARY_AGE = 0
 LIBRARY_CURRENT = 2
@@ -154,7 +154,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -178,10 +178,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/utils/jpgicc
-abs_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/utils/jpgicc
-abs_top_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
-abs_top_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
+abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/jpgicc
+abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/jpgicc
+abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -212,7 +212,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
@@ -239,7 +239,7 @@ AM_CPPFLAGS = -I$(top_builddir)/include
               -I$(top_srcdir)/utils/common -I$(top_builddir)/utils/common
 
 jpgicc_LDADD = $(top_builddir)/src/liblcms2.la -ljpeg -lm
-jpgicc_LDFLAGS = 
+jpgicc_LDFLAGS = -Wl,-Bsymbolic-functions
 jpgicc_SOURCES = jpgicc.c iccjpeg.c ../common/xgetopt.c ../common/vprf.c ../common/utils.h
 man_MANS = jpgicc.1
 EXTRA_DIST = iccjpeg.h $(man_MANS)
--- lcms2-2.2+git20110628.orig/utils/psicc/Makefile
+++ lcms2-2.2+git20110628/utils/psicc/Makefile
@@ -75,13 +75,13 @@ DIST_SOURCES = $(psicc_SOURCES)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -113,7 +113,7 @@ INSTALL_STRIP_PROGRAM = $(install_sh) -c
 JPEGICC_DEPLIBS = -ljpeg -lm
 LCMS_LIB_DEPLIBS = -lm
 LD = /usr/bin/ld -m elf_x86_64
-LDFLAGS = 
+LDFLAGS = -Wl,-Bsymbolic-functions
 LIBOBJS = 
 LIBRARY_AGE = 0
 LIBRARY_CURRENT = 2
@@ -129,7 +129,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -153,10 +153,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/utils/psicc
-abs_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/utils/psicc
-abs_top_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
-abs_top_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
+abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/psicc
+abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/utils/psicc
+abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -187,7 +187,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
@@ -214,7 +214,7 @@ AM_CPPFLAGS = -I$(top_builddir)/include
                -I$(top_srcdir)/utils/common -I$(top_builddir)/utils/common
 
 psicc_LDADD = $(top_builddir)/src/liblcms2.la 
-psicc_LDFLAGS = 
+psicc_LDFLAGS = -Wl,-Bsymbolic-functions
 psicc_SOURCES = psicc.c ../common/xgetopt.c ../common/vprf.c ../common/utils.h
 psicc_MANS = psicc.1
 EXTRA_DIST = $(man_MANS)
--- lcms2-2.2+git20110628.orig/testbed/Makefile
+++ lcms2-2.2+git20110628/testbed/Makefile
@@ -71,13 +71,13 @@ DIST_SOURCES = $(testcms_SOURCES)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -109,7 +109,7 @@ INSTALL_STRIP_PROGRAM = $(install_sh) -c
 JPEGICC_DEPLIBS = -ljpeg -lm
 LCMS_LIB_DEPLIBS = -lm
 LD = /usr/bin/ld -m elf_x86_64
-LDFLAGS = 
+LDFLAGS = -Wl,-Bsymbolic-functions
 LIBOBJS = 
 LIBRARY_AGE = 0
 LIBRARY_CURRENT = 2
@@ -125,7 +125,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -149,10 +149,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/testbed
-abs_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/testbed
-abs_top_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
-abs_top_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
+abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/testbed
+abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/testbed
+abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -183,7 +183,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
@@ -208,7 +208,7 @@ top_srcdir = ..
 AUTOMAKE_OPTIONS = 1.7 foreign
 INCLUDES = -I$(top_builddir)/include -I$(top_srcdir)/include -I$(top_srcdir)/src
 testcms_LDADD = $(top_builddir)/src/liblcms2.la 
-testcms_LDFLAGS = 
+testcms_LDFLAGS = -Wl,-Bsymbolic-functions
 testcms_SOURCES = testcms2.c
 EXTRA_DIST = test1.icc bad.icc toosmall.icc test2.icc \
              test3.icc test4.icc \
--- lcms2-2.2+git20110628.orig/include/Makefile
+++ lcms2-2.2+git20110628/include/Makefile
@@ -76,13 +76,13 @@ HEADERS = $(include_HEADERS)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -114,7 +114,7 @@ INSTALL_STRIP_PROGRAM = $(install_sh) -c
 JPEGICC_DEPLIBS = -ljpeg -lm
 LCMS_LIB_DEPLIBS = -lm
 LD = /usr/bin/ld -m elf_x86_64
-LDFLAGS = 
+LDFLAGS = -Wl,-Bsymbolic-functions
 LIBOBJS = 
 LIBRARY_AGE = 0
 LIBRARY_CURRENT = 2
@@ -130,7 +130,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -154,10 +154,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/include
-abs_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/include
-abs_top_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
-abs_top_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
+abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/include
+abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/include
+abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -188,7 +188,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
--- lcms2-2.2+git20110628.orig/src/Makefile
+++ lcms2-2.2+git20110628/src/Makefile
@@ -99,13 +99,13 @@ DIST_SOURCES = $(liblcms2_la_SOURCES)
 ETAGS = etags
 CTAGS = ctags
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run aclocal-1.11
-AMTAR = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run tar
+ACLOCAL = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run aclocal-1.11
+AMTAR = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run tar
 AR = ar
 AS = as
-AUTOCONF = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoconf
-AUTOHEADER = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run autoheader
-AUTOMAKE = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run automake-1.11
+AUTOCONF = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoconf
+AUTOHEADER = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run autoheader
+AUTOMAKE = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run automake-1.11
 AWK = gawk
 CC = gcc -std=gnu99
 CCDEPMODE = depmode=gcc3
@@ -137,7 +137,7 @@ INSTALL_STRIP_PROGRAM = $(install_sh) -c
 JPEGICC_DEPLIBS = -ljpeg -lm
 LCMS_LIB_DEPLIBS = -lm
 LD = /usr/bin/ld -m elf_x86_64
-LDFLAGS = 
+LDFLAGS = -Wl,-Bsymbolic-functions
 LIBOBJS = 
 LIBRARY_AGE = 0
 LIBRARY_CURRENT = 2
@@ -153,7 +153,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/missing --run makeinfo
+MAKEINFO = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/missing --run makeinfo
 MANIFEST_TOOL = :
 MKDIR_P = /bin/mkdir -p
 NM = /usr/bin/nm -B
@@ -177,10 +177,10 @@ SHELL = /bin/bash
 STRIP = strip
 TIFFICC_DEPLIBS = -ltiff -ljpeg -lz -lm
 VERSION = 2.2
-abs_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/src
-abs_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/src
-abs_top_builddir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
-abs_top_srcdir = /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628
+abs_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/src
+abs_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/src
+abs_top_builddir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
+abs_top_srcdir = /home/till/ubuntu/lcms2/lcms2-2.2+git20110628
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_CXX = g++
@@ -213,7 +213,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
 inline = 
-install_sh = ${SHELL} /home/malex/debian/collab/lcms/lcms2/2.2/2011-06-27/lcms2-2.2+git20110628/install-sh
+install_sh = ${SHELL} /home/till/ubuntu/lcms2/lcms2-2.2+git20110628/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
--- lcms2-2.2+git20110628.orig/libtool
+++ lcms2-2.2+git20110628/libtool
@@ -2,7 +2,7 @@
 
 # libtool - Provide generalized library-building support services.
 # Generated automatically by config.status (lcms2) 2.2
-# Libtool was configured on host pbuilder:
+# Libtool was configured on host till:
 # NOTE: Changes made to this file will be lost: look at ltmain.sh.
 #
 #   Copyright (C) 1996, 1997, 1998, 1999, 2000, 2001, 2003, 2004, 2005,
@@ -272,10 +272,10 @@ finish_eval=""
 hardcode_into_libs=yes
 
 # Compile-time system search path for libraries.
-sys_lib_search_path_spec="/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1 /usr/lib/x86_64-linux-gnu /lib /usr/lib "
+sys_lib_search_path_spec="/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1 /lib /usr/lib "
 
 # Run-time system search path for libraries.
-sys_lib_dlsearch_path_spec="/lib /usr/lib /usr/local/lib /lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu /lib32 /usr/lib32 "
+sys_lib_dlsearch_path_spec="/lib /usr/lib /lib32 /usr/lib32 /usr/local/lib /lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu/mesa-egl /usr/lib/x86_64-linux-gnu/mesa "
 
 # Whether dlopen is supported.
 dlopen_support=unknown
@@ -10264,17 +10264,17 @@ file_list_spec=""
 hardcode_action=immediate
 
 # The directories searched by this compiler when creating a shared library.
-compiler_lib_search_dirs="/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1 /usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/../../.. /usr/lib/x86_64-linux-gnu"
+compiler_lib_search_dirs="/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu /usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1 /usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/../../.."
 
 # Dependencies to place before and after the objects being linked to
 # create a shared library.
-predep_objects="/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/../../../crti.o /usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/crtbeginS.o"
-postdep_objects="/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/crtendS.o /usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/../../../crtn.o"
+predep_objects="/usr/lib/x86_64-linux-gnu/crti.o /usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/crtbeginS.o"
+postdep_objects="/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/crtendS.o /usr/lib/x86_64-linux-gnu/crtn.o"
 predeps=""
 postdeps="-lstdc++ -lm -lgcc_s -lc -lgcc_s"
 
 # The library search path used internally by the compiler when linking
 # a shared library.
-compiler_lib_search_path="-L/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1 -L/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/../../.. -L/usr/lib/x86_64-linux-gnu"
+compiler_lib_search_path="-L/lib/x86_64-linux-gnu -L/usr/lib/x86_64-linux-gnu -L/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1 -L/usr/lib/x86_64-linux-gnu/gcc/x86_64-linux-gnu/4.6.1/../../.."
 
 # ### END LIBTOOL TAG CONFIG: CXX
